torchrl/envs/transforms/transforms.py

```diff
@@ -1264,6 +1266,9 @@ def map_transform(trsf):
         self.transforms = nn.ModuleList(transforms)
         for t in transforms:
             t.set_container(self)
+        self.disable_inv_on_reset = any(
+            [trsf.disable_inv_on_reset for trsf in transforms]
+        )
```

Comment:
nit: There is no need to generate the whole list :)

```suggestion
        self.disable_inv_on_reset = any(trsf.disable_inv_on_reset for trsf in transforms)
```